<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicWinControl</class>
 <widget class="QWidget" name="PicWinControl">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Picture Config</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="txtsizeLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="txtbuttonSetSize">
           <property name="text">
            <string>Set tama√±o</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtsizeX">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtsizeY">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="txtbuttonSetVelocity">
           <property name="text">
            <string>Tiempo foto (ms)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txttextVelocity">
           <property name="text">
            <string>700</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="txtbuttonStartStop">
         <property name="text">
          <string>Comenzar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="txtbuttonStyle">
         <property name="text">
          <string>Poner/Sacar bordes</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="txtbuttonSetPos">
             <property name="text">
              <string>Set pos</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtposX">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtposY">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="txtbuttonShowHideWindow">
           <property name="text">
            <string>Esconder</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>change_size(int,int)</slot>
 </slots>
</ui>
